explorer/client/covidDataExplorer/CovidParams.ts

```diff
@@ -1,12 +1,12 @@
 import { computed, observable, action } from "mobx"
-import { EntityUrlBuilder } from "../ChartUrl"
+import { EntityUrlBuilder } from "../../../charts/ChartUrl"
 import {
     QueryParams,
     strToQueryParams,
     queryParamsToStr
 } from "utils/client/url"
 import { SortOrder } from "charts/SortOrder"
-import { oneOf, uniq, intersection } from "../Util"
+import { oneOf, uniq, intersection } from "../../../charts/Util"
```

Comment:
Might be better to use`"charts/..."` rather than `"../../../charts/.." ?

Reply:
Yes. Sometimes when VS Code auto renames imports after a file move it does funky things :)

Reply:
There were a few of these. Thanks